math/src/elliptic_curve/short_weierstrass/curves/bls12_381/curve.rs

```diff
@@ -8,6 +9,25 @@ use crate::{
 pub type BLS12381FieldElement = FieldElement<BLS12381PrimeField>;
 pub type BLS12381TwistCurveFieldElement = FieldElement<Degree2ExtensionField>;
 
+/// Cube Root of Unity in the base field of BLS12-381.
+pub const CUBE_ROOT_OF_UNITY_G1: BLS12381FieldElement = BLS12381FieldElement::from_hex_unchecked(
+    "5f19672fdf76ce51ba69c6076a0f77eaddb3a93be6f89688de17d813620a00022e01fffffffefffe",
+);
+
+/// Cube Root of Unity in the degree-2 extension field of BLS12-381
+pub const CUBE_ROOT_OF_UNITY_G2: BLS12381TwistCurveFieldElement = BLS12381TwistCurveFieldElement::const_from_raw([FieldElement::<BLS12381PrimeField>::from_hex_unchecked("1a0111ea397fe699ec02408663d4de85aa0d857d89759ad4897d29650fb85f9b409427eb4f49fffd8bfd00000000aaac"), FieldElement::<BLS12381PrimeField>::from_hex_unchecked("0")]);
+
+pub const CUBE_ROOT_OF_UNITY_G2_SQUARE: BLS12381TwistCurveFieldElement =
+    BLS12381TwistCurveFieldElement::const_from_raw([
+        FieldElement::<BLS12381PrimeField>::from_hex_unchecked(
+            "5f19672fdf76ce51ba69c6076a0f77eaddb3a93be6f89688de17d813620a00022e01fffffffefffe",
+        ),
+        FieldElement::<BLS12381PrimeField>::from_hex_unchecked("0"),
+    ]);
+
+/// Seed value for BLS-12 381 curve.
```

Comment:
I got the value from [here](https://hackmd.io/@yelhousni/bls12_subgroup_check). There it is mentioned as seed.

Reply:
Ah I see. For the sake of clarity, let's cite Construction 6.6 of the taxonomy of elliptic curves [paper](https://theory.stanford.edu/~dfreeman/papers/taxonomy.pdf) and mention that `SEED` is $-x$ where $x$ is the parameter of the Construction 6.6 for $k=12$. And add the formulas for the subgroup order and the field size in terms of $x$.
Check out this [site](https://fkunstner.github.io/latex-to-utf8/) to convert latex to ut8 and produce more legible formulas in code comments, (like x⁴ − x² + 1 instead of x^4 - x^2 + 1)